src/libsystemd/sd-bus/bus-control.c

```diff
@@ -876,9 +876,10 @@ _public_ int sd_bus_get_owner_creds(sd_bus *bus, uint64_t mask, sd_bus_creds **r
 int bus_add_match_internal(
                 sd_bus *bus,
                 const char *match,
-                uint64_t *ret_counter) {
+                uint64_t *ret_counter,
+                uint64_t timeout_usec) {
```

Comment:
maybe swap the order of these two args. while the rules are a bit murky we usually put arguments that are output only after those which are input only